integration/ig/k8s/trace_dns_test.go

```diff
@@ -175,7 +175,7 @@ func TestTraceDns(t *testing.T) {
 					PktType:    "HOST",
 					DNSName:    "nodomain.fake.test.com.",
 					QType:      "A",
-					Rcode:      "NXDomain",
+					Rcode:      "Non-Existent Domain",
```

Comment:
Should the width of the column be adapted?

pkg/gadgets/trace/dns/types/dns.go:
```
        Rcode      string        `json:"rcode,omitempty" column:"rcode,minWidth:8"`
```

Reply:
I changed it to 12.